src/muya/lib/assets/styles/index.css

```diff
@@ -126,6 +126,9 @@ figure[data-role="HTML"] .ag-html-preview {
 figure[data-role="HTML"] .ag-html-preview > pre{
   border: 0;
   background: inherit;
+  padding: 0 .5rem;
```

Comment:
Thanks @mdogadailo for the PR. Could you please use `1em` padding like code blocks.

Reply:
Hi @fxha, updated.